docs/admin/misc_configuration_tips.md

```diff
@@ -65,4 +65,19 @@ If you want to add some specific JVM settings, you can do so by using the `JDK_J
 ```yaml
     environment:
       - JDK_JAVA_OPTIONS=-Xms1024m -Xmx2048m
-```
\ No newline at end of file
+```
+
+## Launch a second instance of stellio 
+
+The csr.env file contains all the configuration needed to run a second instance of stellio.
```

Comment:
I don't see the file in the commit, normal?

Reply:
Yes, this is stellio-docs